### PR TITLE
llvd: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/llvd.md
Comment (einverne) [inline pages/linux/llvd.md:8]: `course-slug` seems to be the part of the URL of the course, it should be wrapped with `{{}}`.

```suggestion
`llvd -c {{course-slug}} --cookies`
```

By the way, can you provide an example of `course-slug`?

**Review by einverne — CHANGES_REQUESTED** [review]
@suyashbhawsar thank you for the new page. I left a comment for you to take a look at.

**Review by einverne — APPROVED** [review]

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:10]:
```suggestion
- Download a course [p]ath with [t]hrottling between 10 to 30 seconds:
```
I think mentioning seconds is essential in this command description as it gives context to the throttling flag.

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:6]:
```suggestion
- Download an [c]ourse using cookie-based authentication:
```
I think mentioning why we use the cookies flag in the command description would be better.

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:9]:
```suggestion

- Download a course at a specific [r]esolution:

`llvd -c {{course-slug}} -r 720`

```
I think this command will be helpful too on this page.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution @suyashbhawsar, I have some suggestions that need to be addressed. 

Try to use mnemonics whenever possible in command descriptions for single-character command flags.

**Comment by suyashbhawsar** [comment]
> @suyashbhawsar thank you for the new page. I left a comment for you to take a look at.

Thanks for the suggestions!

**Comment by suyashbhawsar** [comment]
@kbdharun Are the tldr pages in Linux directories accessible for the other Platforms using the tldr? The common directory already have 1000 files & I'm not able to create another one so, I just created the file here.

**Comment by suyashbhawsar** [comment]
> Thanks for your contribution @suyashbhawsar, I have some suggestions that need to be addressed.
> 
> Try to use mnemonics whenever possible in command descriptions for single-character command flags.

Thanks, I'll keep that in mind.

**Comment by kbdharun** [comment]
> @kbdharun Are the tldr pages in Linux directories accessible for the other Platforms using the tldr?

Yes, tldr-pages directories are accessible on all platforms, and you can access other platform commands too.
For example, you can access `winget` command in linux and it works without any additional steps, for some commands like `uname` there are separate pages on each platform. You can access it using these flags `--linux, --osx,  --sunos, --windows` in our default npm client to view platform-specific pages.

> The common directory already has 1000 files & I'm not able to create another one so, I just created the file here.

I think you are mentioning this (Ref image 1), GitHub truncates entries in directories to a maximum of 1000, in simple terms GitHub displays only 1000 files in a directory it doesn't restrict adding files to the directory. **I noticed as you mentioned the page is cross-platform** *so it must be placed under `common`.* If possible try to use Git and clone your fork and move the file from `linux` to `common`. Or else feel free to comment here, I will do it for you.

Ref image 1:
image

> Sorry, for the late reply, I had some work today, I will review this PR now.

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:22]:
```suggestion

```

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:23]:
```suggestion
```

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:24]:
```suggestion
```

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:25]:
```suggestion
```

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:26]:
```suggestion
```

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:27]:
```suggestion
```

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:28]:
```suggestion
```

**Review by kbdharun — COMMENTED** [review]
I have some suggestions we can remove the last two commands as they can't be run individually (I tested llvd by installing it on my PC). Also, we have already added new command descriptions to this page about those flags so I think it isn't necessary.

Tip: Try to add all suggestions to batch in this PRs file tab and make a single commit.

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:14]:
```suggestion
- Download a course with [c]aptions (subtitles):
```

**Comment by suyashbhawsar** [comment]
> I have some suggestions we can remove the last two commands as they can't be run individually (I tested llvd by installing it on my PC). Also, we have already added new command descriptions to this page about those flags so I think it isn't necessary.
> 
> Tip: Try to add all suggestions to batch in this PRs file tab and make a single commit.

These are options & not complete commands. Can you tell me how should I write it? At least the caption flag is important. You can try the option with any of the first two commands.

Also, how can I write suggestions like the content of cookies.txt?

**Comment by kbdharun** [comment]
> These are options & not complete commands. Can you tell me how should I write it? At least the caption flag is important. 

From, what I found the `llvd --caption`,  and `llvd --resolution` didn't work individually for me in Linux, can you give me some insights on whether these commands work individually?

> You can try the option with any of the first two commands.

Since we added examples for command usage separate options wouldn't be necessary. Also, user might get confused thinking it is a default option setting.

> Also, how can I write suggestions like the content of cookies.txt?

I think you are talking about the command description, mentioning details about flags is optional, for example, the first command tells the user that cookie-based authentication is done to download the video as you used the `--cookie` flag in commands so describing it would be nice.

**Comment by suyashbhawsar** [comment]
> I think you are talking about the command description, mentioning details about flags is optional, for example, the first command tells the user that cookie-based authentication is done to download the video as you used the `--cookie` flag in commands so describing it would be nice.

Yeah. But, is there a separate syntax to write suggestions?

**Review by kbdharun — APPROVED** [review]
The page LGTM, now we just need to move it to the `common` directory.
> **I noticed as you mentioned the page is cross-platform** _so it must be placed under `common`._ If possible try to use Git and clone your fork and move the file from `linux` to `common`. Or else feel free to comment here, I will do it for you.

File: pages/linux/llvd.md
Comment (kbdharun) [inline pages/linux/llvd.md:14]: Just a small suggestion
```suggestion
- Download a course with captions (subtitles):
```
Here, captions isn't a valid flag as `--ca` doesn't work.

> Yeah. But, is there a separate syntax to write suggestions?

You could refer to our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) for information on writing command descriptions, syntax and other information.

Comment (suyashbhawsar) [reply pages/linux/llvd.md:14]: > Here, captions isn't a valid flag as `--ca` doesn't work.

Try `-ca`

Comment (kbdharun) [reply pages/linux/llvd.md:14]: Yep, ca works. I will resolve this conversation then.
Ref. 
image

**Comment by kbdharun** [comment]
@suyashbhawsar was this PR closed by accident?

**Comment by suyashbhawsar** [comment]
> @suyashbhawsar was this PR closed by accident?

Yup, I renamed the branch. Can I revert it back?

**Comment by suyashbhawsar** [comment]
@kbdharun why isn't it merging?

**Comment by suyashbhawsar** [comment]
> Merging is done manually by maintainers/collaborators when there are 2 or more approvals. We can't merge a page which doesn't have 2 approvals. This page is good to go, I just updated the directory to common. Thanks for this massive PR heart.

Oh… I thought it was an automatic process.

**Comment by kbdharun** [comment]
> > Merging is done manually by maintainers/collaborators when there are 2 or more approvals. We can't merge a page which doesn't have 2 approvals. This page is good to go, I just updated the directory to common. Thanks for this massive PR heart.
> 
> Oh… I thought it was an automatic process.

It used to be an automatic process, but It was changed to manual because in some PRs we need opinions and reviews from multiple maintainers but when 2 approvals are reached GitHub automatically merged it even with some unresolved conversations. So we changed to a manual process.

**Pull request merged**
